widget/embedded/src/hooks/useSearchCustomTokens/useSearchCustomTokens.ts

```diff
@@ -0,0 +1,74 @@
+import type { Token } from 'rango-sdk';
+
+import { useCallback, useRef, useState } from 'react';
+
+import { httpService } from '../../services/httpService';
+import { useAppStore } from '../../store/AppStore';
+import { createAssetKey } from '../../store/utils/wallets';
+import { debounce } from '../../utils/common';
+
+import { DEBOUNCE_DELAY } from './useSearchCustomTokens.constants';
+
+export function useSearchCustomTokens() {
+  const abortController = useRef<AbortController | null>(null);
+  const { customTokens } = useAppStore();
+  const [loading, setLoading] = useState(false);
+  const [tokens, setTokens] = useState<Token[]>([]);
+  const [error, setError] = useState<string | null>();
```

Comment:
Isn't better to fix state initialization for the error state?
```suggestion
  const [error, setError] = useState<string | null>(null);
```